benchmarks/dynamo/common.py

```diff
@@ -55,6 +55,8 @@
 
 try:
     import torch_xla.core.xla_model as xm
+
+    device = xm.xla_device()
```

Comment:
what's the purpose of calling xla_device()?

i assume if it failed it would throw a different exception so catching importerror isn't enough, also its odd to bind it to a name.  is it enough to just do the import here and do the device call somewhere later at first use?

Reply:
the actual bug is discussed in https://github.com/pytorch/xla/issues/4174. I think it required pytorch/xla to registered something to the pytorch so backward can run correctly(when there is a cpu backward was run first). 

The reason that old workaround works is we used to eagerly init the runtime but now that doesn't happen until actual device is being init. I need to root cause this issue...

Reply:
Init the device is not ideal as it actually make it impossible to run more than one tests at a same time on TPU, since the main process init the device outside of the `spawn` function and keep the device.

I tried a few other approach and they didn't work. I decided it is better to at least make training runable and fix this later in nightly

Reply:
yea, let's make sure the script still works when XLA is missing.

Reply:
it should, since the device init only happens when torch_xla import succed.

Reply:
Let me try to dig into this issue a bit more this afternoon and see if I can workaround it.

Reply:
I think a few things may make it better
1. have a API to explicitly initialize the xla runtime and call that rather than 'xla_device'. This can avoid some confusion.
2. or figure out the root cause :) 

But I'm ok to stamp the PR now to unblock. Unless @wconstab has other thoughts.

Reply:
Thanks Shunting, I added a API to explicitly init runtime. Let me update it.